### DISCUSSION TITLE
notifications badge and gestures transitions

### DISCUSSION THREAD

**Discussion by Dewet-sa** [tanujnotes/Olauncher#218] [Ideas]
![Picsart_22-08-26_10-02-14-696.jpg](https://user-images.githubusercontent.com/69904344/186857718-9d39b428-ff7e-40f8-8ce2-e41d5aa8014d.jpg) 

1. home screen notifications icon badges beside the date or some sort of notification on the app names on both home screen and app drawer 

2.  L/R slide transition animation for the LEFT/RIGHT swipe gesture and mybe up slide transition for opening https://youtu.be/0s6x3Sn4eYo

**Comment by tanujnotes** [comment]
1. No plans to add this for now.
2. The link you provided works for activities between the same app, not when opening another app.

I want to add app opening animations. Currently there's no timeline though.

**Reply by hayribakici** [reply]
@tanujnotes Would you accept pull requests regarding point number 1?

**Reply by tanujnotes** [reply]
@hayribakici I honestly don't see that much of a need for it.
